### PR TITLE
164489894 User should be able like and dislike articles.

### PR DESCRIPTION
**Pull request by cop1fab**
**What does this PR do?**

This PR allows a user to like and/or dislike articles. 

**Description of Task to be completed?**

Users can like or dislike articles, both authored by the user who is liking and those not authored by others.

**How should this be manually tested?**

After pulling run `npm install`
login then read a single article, you will see buttons at the bottom right of the pages, to allow you to either like or dislike an article.  

**What are the relevant pivotal tracker stories?**

[#164489894](https://www.pivotaltracker.com/story/show/164489894)

**Screenshots (if appropriate)**
image

### CODE REVIEW

File: src/actions/voteArticle.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+const hashids = new Hashid('', 10);
+const BACKEND_URL = 'https://badass-ah-backend-staging.herokuapp.com';
+
+export const likeArticle = id => async (dispatch) => {
+  const url = `${BACKEND_URL}/api/articles/${hashids.decode(id)}/like`;
+  const url2 = `${BACKEND_URL}/api/articles/${hashids.decode(id)}`;
+  try {
+    const Like = await axios.post(url, '', config);
+    const single = await axios.get(url2);
```
Comment (nakiwuge) [inline src/actions/voteArticle.js]: Add a new line after this line

File: src/actions/voteArticle.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+};
+
+export const dislikeArticle = id => async (dispatch) => {
+  const url = `${BACKEND_URL}/api/articles/${hashids.decode(id)}/dislike`;
+  const url2 = `${BACKEND_URL}/api/articles/${hashids.decode(id)}`;
+  axios.post(url, '', config);
+
+  try {
+    const Dislike = await axios.post(url, '', config);
+    const single = await axios.get(url2);
```
Comment (nakiwuge) [inline src/actions/voteArticle.js]: Add a new line after this line

File: src/assets/scss/_article.scss [hunk 3]
```diff
@@ -231,17 +264,61 @@
...
+        position: fixed center;
+        right: 15px;
+        bottom: 15px;
+        z-index: 100;
+    }
+    .c-like-grid{
+        display: grid;
+        grid-template-columns: 0fr 0fr;
+        grid-gap: 10px;
+        // :hover{
```
Comment (nakiwuge) [inline src/assets/scss/_article.scss]: remove commented code

File: src/assets/scss/_article.scss [hunk 4]
```diff
@@ -231,17 +264,61 @@
...
+    .turnBlack{
+        color: black!important;
+    }
+    .changeColor{
+        color: red;
+    }
+    .btn-heart {
+        font-family: $rokitFont;
+        border-radius: 10px;
+        // color: #000;
```
Comment (nakiwuge) [inline src/assets/scss/_article.scss]: remove commented code

File: src/assets/scss/_article.scss [hunk 5]
```diff
@@ -196,28 +224,33 @@
...
         position: absolute;
         right: 0;
         top: 0;
-        .G-displayError{
+
+        .G-displayError {
             position: relative;
             margin-top: 100px;
+
             // transform: translatex(100%);
```
Comment (nakiwuge) [inline src/assets/scss/_article.scss]: remove commented code

**Review by nakiwuge — APPROVED** [review]

**Review by itsgratien — APPROVED** [review]

**Pull request merged**